### DISCUSSION TITLE
Deplyment of new backpack for Laravel app

### DISCUSSION THREAD

**Discussion by AlonMoshe** [Laravel-Backpack/community-forum#1044] [Q&A (Help)]
Hi There,

After purchasing a license for a one project app, and after deploying in production, it takes the app an unusually long time (around 2 min) to load anything, everything works well, except the time it takes to load anyone page.
 
I have already tried the following:

1. In .env:

```APP_ENV=production```
````APP_DEBUG=false.````

2. Caching:
```
php artisan config:cache
php artisan route:cache 
php artisan view:cache 
php artisan event:cache 
```

3. Optimizing Composer Autoloader:
`composer install --optimize-autoloader --no-dev`

None of the above helped.

Before installing backpack pro, everything was ok. 

I appreciate any suggestions anyone may have.

Many thanks.

Alon

**Comment by pxpm** [comment]
@AlonMoshe I don't think there is anything in backpack/pro that would make it slower. 

My guess is that you are not caching your assets with basset, so when you open a page, basset need to fetch all those assets first. 

Add to your deployment script: 

```php
php artisan basset:clear
php artisan basset:cache
```

This should internalize your assets and they will be loaded from local server directly. 

Let me know if that helps. 

Cheers

**Comment by AlonMoshe** [comment]
Thank  you @pxpm ,

I have just tried that, but it didn't help. 

My hunch now is that it has nothing to do with Backpack, but rather that it may be a server side issue. 

Also, my application is querying a database that is on a different server, I will need to make sure that the long delay isn't something related to this.  

In any case, if you have any suggestions, they are welcome.

Thanks,

Alon.
